CK2Plus/history/characters/castillan.txt

```diff
@@ -2328,6 +2328,11 @@
 	1136.7.29 = {
 		birth = yes
 	}
+	1138.1.1 = {
+		effect = {
+			add_betrothal = 210506
```

Comment:
This is kind of fucked, but at the age of 1 she was betrothed to the 24 year old Ramon Berenguer.